src/lib/components/SplitBlock.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+</script>
+
+<div class="split-block">
+  <div class="start">
+    <slot name="start" />
+  </div>
+  <div class="end">
+    <slot name="end" />
+  </div>
+</div>
+
+<style lang="scss">
+  @use "../styles/mixins/media";
+
+  .split-block {
+    @include media.min-width(large) {
+      display: flex;
+      flex-direction: row;
+      gap: var(--padding-4x);
+    }
+  }
+
+  .start,
+  .end {
+    flex: 1;
+  }
+
+  // line separator
+  .start {
+    position: relative;
+
+    &::after {
+      content: "";
+      display: block;
+      height: 1px;
+      background: var(--line);
+      margin: var(--padding-2x) 0;
```

Comment:
If we use flex, shuold we add the separator as an html element instead?

Otherwise we are mixing `gap` and `margin` for the spacings.

Reply:
I mean that the UI has some spacings between the elements inside.

Some spacing comes from a `gap` rule, other from a `margin`. But all could be done with the `gap`, right?

Reply:
I find using flex/grid just to have `gap` is not optimal, so on mobile it is `display: block` (not flex). Would you prefer to use gap here and place the separator line in the center of it?

Reply:
Why isn't it optimal?

Why do we need to change from flex to box?

I feel that on mobile changing the flex direction and gap should be enough, isn't it?

Reply:
Ok, I'm convinced. @lmuntaner please re-review.